notebooks/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>I think it would be good to put <code>data_path</code> in this cell:</p><p><code></code><code></p><p>data_path = os.path.join("..", "..", "tests", "resources", "deeprec", "slirec")</p><p></code><code></code></p><p>or if you want to use the TemporaryDirectory:</p><p><code></code><code></p><p>tmp_dir = TemporaryDirectory()</p><p>data_path = tmp_dir.name</p><p></code><code></code></p><p>if we move data_path to this cell, we can make add <code>parameters</code> label here and inject different values from papermill</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1010/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Got it.